src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -61,6 +94,14 @@ export const UserOrcid = ({ user }: UserOrcidProps) => {
           method: 'PATCH',
           data: { orcid },
         });
+        const orcidCredentials = getOrcidCredentials(history.location.search, orcidInfoResponse.data.id);
+        if (!orcidCredentials) {
+          dispatch(setNotification({ message: t('feedback.error.storing_orcid_credentials'), variant: 'success' }));
+        } else {
+          postOrcidCredentials(orcidCredentials).catch((_fail) => {
+            dispatch(setNotification({ message: t('feedback.error.storing_orcid_credentials'), variant: 'success' }));
+          });
+        }
         if (isSuccessStatus(addOrcidResponse.status)) {
           dispatch(setNotification({ message: t('feedback.success.update_orcid'), variant: 'success' }));
           cristinPersonQuery.refetch();
```

Comment:
Vi trenger sikkert bare å prøve dette når tilkobling faktisk gikk bra? Lurer på om vi kan droppe suksessmelding her også, siden dette skjer "skjult" for brukeren. `apiRequest2` er ganske ny og ikke mye brukt enda, så vi burde dobbeltsjekke at try-catch virker som forventet der

```suggestion
        if (isSuccessStatus(addOrcidResponse.status)) {
          dispatch(setNotification({ message: t('feedback.success.update_orcid'), variant: 'success' }));
          cristinPersonQuery.refetch();
          
          const orcidCredentials = getOrcidCredentials(history.location.search, orcidInfoResponse.data.id);
            if (orcidCredentials) {
              try {
                await postOrcidCredentials(orcidCredentials);
              } catch {
                dispatch(setNotification({ message: t('feedback.error.storing_orcid_credentials'), variant: 'error' }));
              }
           }
```

Reply:
Endte opp med å sette alle setNotifications helt nederst før jeg så kommentaren din. Nå er de litt samlet sammen.